src/parser/getCSSRules.ts

```diff
@@ -30,6 +30,13 @@ export function getCSSRules(doc: Document) {
       .filter((rule, index, array) => array.length > 1 && rule.trim())
       // at this point we have hopefully an array of rules `body { style code... `
       .forEach((rule) => {
+        // if there is more than one opening bracket and the rule starts with '@', it is likely
+        // a nested at-rule like @media, @supports, @scope, etc. Ignore these as the code below
+        // can not handle it.
+        if ((rule.match(/{/g) || []).length > 1 && rule.trimStart().startsWith('@')) {
+          return;
+        }
```

Comment:
This is our current browserlists for this release:
```
chrome >= 64
safari >= 11
firefox >= 58
```

trimStart doesn't  fit.
It doesn't matter how we ended up on this list of browsers, we have this for 6.0 and now is not the moment to change them.
Can you use trim? it should make the job the same way in our case that we are testing 'startsWith'

Reply:
I had `.trim` before but thought it would be slightly inefficient, I will revert to `.trim`.

Reply:
I doubt efficiency is at stake here, this code is already bad.